packages/react-ui/accessibility/src/FocusGrid.js

```diff
@@ -7,11 +7,11 @@
  * @flow
  */
 
-import type {KeyboardEvent} from 'react-events/src/dom/Keyboard';
+import type {KeyboardEvent} from 'react-ui/events/src/dom/Keyboard';
 
 import React from 'react';
 import {tabFocusableImpl} from './TabbableScope';
-import {useKeyboard} from 'react-events/keyboard';
+import {useKeyboard} from '../../events/keyboard';
```

Comment:
Oops this is the non-type import that could get inlined

Reply:
This is okay. I plan on doing a follow up where I add the a11y components to build and will revise it there.